docs/paper.bib

```diff
@@ -0,0 +1,258 @@
+	number		 = {1},
+	pages		 = {201--231},
+	year		 = {2022},
+	publisher	 = {The Institute of Mathematical Statistics and the Bernoulli Society},
+	doi          = {10.1214/21-ejs1957}
+}
+@Article{Gneiting:2011,
+	title		 = {Comparing density forecasts using threshold-and quantile-weighted scoring rules},
+	author		 = {Gneiting, Tilmann and Ranjan, Roopesh},
+	journal		 = {Journal of Business & Economic Statistics},
+	volume		 = {29},
+	number		 = {3},
+	pages		 = {411--422},
+	year		 = {2011},
+	publisher	 = {Taylor & Francis},
+	doi          = {10.1198/jbes.2010.08110}
+}
+@misc{Taggart:2022d,   
+    title    	 = {Assessing calibration when predictive distributions have discontinuities.},   
+    url 		 = {http://www.bom.gov.au/research/publications/researchreports/BRR-064.pdf},   
+    author 		 = {Taggart, Robert},   
+    year 		 = {2022},   
+    note 		 = {Accessed on September 9, 2023} 
+}
+@misc{loveday2023userfocused,
+      title={A User-Focused Approach to Evaluating Probabilistic and Categorical Forecasts}, 
+      author={Nicholas Loveday and Robert Taggart and Mohammadreza Khanarmuei},
+      year={2023},
+      eprint={2311.18258},
+      archivePrefix={arXiv},
+      primaryClass={stat.AP}
+}
+@article{nipen2023verif,
+  title={Verif: A weather-prediction verification tool for effective product development},
+  author={Nipen, Thomas N and Stull, Roland B and Lussana, Cristian and Seierstad, Ivar A},
+  journal={Bulletin of the American Meteorological Society},
+  volume={104},
+  number={9},
+  pages={E1610--E1618},
+  year={2023},
+  publisher={American Meteorological Society},
+  doi={10.1175/bams-d-22-0253.1}
+}
+@article{dimitriadis2021stable,
+  title={Stable reliability diagrams for probabilistic classifiers},
+  author={Dimitriadis, Timo and Gneiting, Tilmann and Jordan, Alexander I},
+  journal={Proceedings of the National Academy of Sciences},
+  volume={118},
+  number={8},
+  pages={e2016191118},
+  year={2021},
+  publisher={National Acad Sciences},
+  doi={10.1073/pnas.2016191118}
+}
+@article{griffiths2021circular,
+  title={Circular Flip-Flop Index: quantifying revision stability of forecasts of direction},
+  author={Griffiths, Deryn and Loveday, Nicholas and Price, Benjamin and Foley, Michael and McKelvie, Alistair},
+  journal={Journal of Southern Hemisphere Earth Systems Science},
+  volume={71},
+  number={3},
+  pages={266--271},
+  year={2021},
+  publisher={CSIRO Publishing},
+  doi={10.1071/es21010}
+}
+@misc{loveday2024jive,
+      title={The Jive Verification System and its Transformative Impact on Weather Forecasting Operations}, 
+      author={Nicholas Loveday and Deryn Griffiths and Tennessee Leeuwenburg and Robert Taggart and Thomas C. Pagano and George Cheng and Kevin Plastow and Elizabeth Ebert and Cassandra Templeton and Maree Carroll and Mohammadreza Khanarmuei and Isha Nagpal},
+      year={2024},
+      eprint={2404.18429},
+      archivePrefix={arXiv},
+      primaryClass={physics.ao-ph}
+}
```

Comment:
DOI for the Jive arXiv preprint: https://doi.org/10.48550/arXiv.2404.18429

(The JOSS PDF I rendered yesterday showed https://arxiv.org/abs/2404.18429, so should probably update to DOI)

Reply:
I'm not sure if arXiv DOIs work differently, I've updated the bib file, but there is the risk that something won't work in the PDF.

Updated in https://github.com/nci/scores/pull/398

Reply:
Looks like the DOI is now okay in the updated version in my pull request